packages/@atjson/source-html/test/converter-test.ts

```diff
@@ -402,6 +402,28 @@ describe("@atjson/source-html", () => {
       });
     });
 
+    test("protocol-relative iframe embeds", () => {
```

Comment:
What am i missing here -- it seems to be that this test isn't actually testing your change EG not actually testing that the https is added, no?

Reply:
Oh, it's testing that it doesn't throw an error 🙃

Reply:
@dkorenblyum I added some actual tests for this